### DISCUSSION TITLE
Is there a way to set flake8 errors to be shown as hints?

### DISCUSSION THREAD

**Discussion by lucatpa** [microsoft/pyright#1954] [Q&A]
Hi, I'm using the option `python.linting.flake8Enabled` but I noticed all flake8 issues are shown as errors, and since they are usually only formatting issues, I rather have them as hints, or warning at the most. Is there a way to achieve this?

**Comment by erictraut** [comment]
Pyright and flake8 are completely separate tools. Pyright doesn't pay any attention to settings used by flake8 and vice versa. You'd need to talk to the maintainers of flake8 for this question.
